### ISSUE TITLE
No provider of glFenceSync found

### ISSUE THREAD

**Issue by sim6** [flathub/in.cinny.Cinny#52]
Running on Librem5 crash with:
```
No provider of glFenceSync found.  Requires one of:
    Desktop OpenGL 3.2
    GL_ARB_sync
    OpenGL ES 3.0
    GL_APPLE_sync
```

**Comment by sim6** [comment]
Works if I disable GPU acceleration (device=dri) via Flatseal from pureos.store

**Comment by camelCaseNick** [comment]
This is not an issue directly with Cinny or Cinny's flatpak. It was reported upstream to Tauri https://github.com/tauri-apps/tauri/issues/7334, and its upstream GTK https://gitlab.gnome.org/GNOME/gtk/-/issues/5749. It looks like, it should be fixed by GTK version 3.24.39.
